app/scripts/modules/core/src/reactShims/react.uirouter.tsx

```diff
@@ -8,6 +8,15 @@ import './react.uirouter.css';
 export const REACT_UIROUTER = 'spinnaker.core.react.uirouter';
 const hybridModule = module(REACT_UIROUTER, ['ui.router']);
 
+const bindResolves = (WrappedComponent: any) =>
+class BindResolvesToProps extends React.Component<any, any> {
```

Comment:
should be able to remove the second `any` after https://github.com/spinnaker/deck/pull/3908